docs/src/symbolic_regression.md

```diff
@@ -1,10 +1,30 @@
 # Symbolic Regression
 
-!!! info
-    There is also the excellent [SymbolicRegression.jl](https://github.com/MilesCranmer/SymbolicRegression.jl) package, which will be interfaced soon(ish).
-
 Using [sparse regression](@ref sparse_optimization) limits the discovery to a generalized linear model where it is assumed that the nonlinear [basis](@ref) can capture the underlying function properly. Another approach is to use a general expression tree, which commonly encodes the function to discover as a binary tree where the nodes represent unary or binary operators acting on their children. `DataDrivenDiffEq` includes the following symbolic regression algorithms.
 
+## SymbolicRegression
+
+!!! warning
+    This feature requires the explicit loading of [SymbolicRegression.jl](https://github.com/MilesCranmer/SymbolicRegression.jl) in addition to `DataDrivenDiffEq`. It will _only_ be useable if used like
+    ```julia
+    using DataDrivenDiffEq
+    using SymbolicRegression
+    ```
+
+`DataDrivenDiffEq` provides an interface to [SymbolicRegression.jl](https://github.com/MilesCranmer/SymbolicRegression.jl) to `solve` a [`DataDrivenProblem`](@ref):
+
+```julia
+# Define the symbolic regression options
+opts = Options(binary_operators = (+, *),unary_operators = (exp, sin), maxdepth = 1, progress = true, verbosity = 0)
```

Comment:
Make this just be named to be the SymbolicRegressionAlg or something?

Reply:
What about `EQSeach` ?

Reply:
Sure, though `SymbolicRegressionJL` might make it more clear what the actual alg is?

Reply:
Tbh, I don't care as much :smile: . I just think that `EQSearch` is similar to the `Equationsearch` provided by SymbolicRegression.jl . But right now, I have problems getting the tests to run locally due to the distributed character of SymbolicRegression.